.travis.yml

```diff
@@ -8,8 +8,9 @@ install:
     - pip install coveralls
     - npm install -g markdownlint-cli
 script:
-    - pytest tabpy-server/server_tests/ --cov=tabpy-server/tabpy_server
-    - pytest tabpy-tools/tools_tests/ --cov=tabpy-tools/tabpy_tools --cov-append
+    - export PYTHONPATH=./tabpy-server:./tabpy-tools:$PYTHONPATH
```

Comment:
1) why do we need this?
2) if we do need it, let's keep it in [env](https://docs.travis-ci.com/user/environment-variables/) or use `utils/set_env.sh`

Reply:
We need it for test code to be able to discover and import tabpy_srver and tabpy_tools packages. Changing to calling script...